config/custom-environment-variables.yaml

```diff
@@ -18,6 +18,9 @@ auth:
     # A password used for encrypting stored secrets.
```

Comment:
can we add comment for this also used to encrypt user oath?

Reply:
Sure